ledgerjs-fio/src/interactions/signTransaction.ts

```diff
@@ -0,0 +1,133 @@
+import type {HexString, ParsedTransaction, ParsedTransferFIOTokensData, Uint8_t, ValidBIP32Path} from "types/internal"
+
+import {InvalidDataReason} from "../errors"
+import type {SignedTransactionData, Version} from "../types/public"
+import {HARDENED, Transaction} from "../types/public"
+import {chunkBy} from "../utils/ioHelpers"
+import {validate} from "../utils/parse"
+import {uint8_to_buf} from "../utils/serialize"
+import {
+    buf_to_hex,
+    date_to_buf,
+    hex_to_buf,
+    path_to_buf,
+    uint16_to_buf,
+    uint32_to_buf,
+    uint64_to_buf,
+} from "../utils/serialize"
+import {INS} from "./common/ins"
+import type {Interaction, SendParams} from "./common/types"
+import {ensureLedgerAppVersionCompatible} from "./getVersion"
```

Comment:
Some unused imports and also the first one should be imported by a relative path as well.
```suggestion
import {InvalidDataReason} from "../errors"
import type {HexString, ParsedTransaction, ParsedTransferFIOTokensData, Uint8_t, ValidBIP32Path} from "../types/internal"
import type {SignedTransactionData, Version} from "../types/public"
import {HARDENED} from "../types/public"
import {chunkBy} from "../utils/ioHelpers"
import {validate} from "../utils/parse"
import {uint8_to_buf} from "../utils/serialize"
import {
    buf_to_hex,
    date_to_buf,
    path_to_buf,
    uint16_to_buf,
    uint32_to_buf,
    uint64_to_buf,
} from "../utils/serialize"
import {INS} from "./common/ins"
import type {Interaction, SendParams} from "./common/types"
import {ensureLedgerAppVersionCompatible} from "./getVersion"
```